LondonBritishLibrary/orient/BLorient732.xml

```diff
@@ -30,20 +31,93 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 309</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
+                        <msItem xml:id="ms_i1">
+                          <locus from="4r"></locus>
+                          <title type="complete" ref="LIT2456Tergwa"></title>
+                          <textLang mainLang="gez"></textLang>
+
+                          <msItem xml:id="ms_i1.1">
+                            <locus from="4r"></locus>
+                            <title type="complete" ref="LIT2456Tergwa#Matthew"></title>
+                            <textLang mainLang="gez"></textLang>
+
+                            <msItem xml:id="ms_i1.1.1">
+                              <locus from="4r"></locus>
+                              <title type="complete" xml:lang="en">Introduction from Ephraim Syrus</title>
+                              <textLang mainLang="gez"></textLang>
+                            </msItem>
+                            <msItem xml:id="ms_i1.1.2">
+                              <locus from="13v"></locus>
+                              <title type="complete" xml:lang="en">Commentary on the Gospel of Matthew</title>
+                              <textLang mainLang="gez"></textLang>
+                              <incipit xml:lang="gez">
+                                ንጽሕፍ፡ መቅድመ፡ ወንጌል፡ ዘማቴዎስ፡ ሐዋርያ፡ ቡሩክ፡ አስተጋቢአነ፡ እምአብርክሲስ፡ ወእምትርጓሜሆሙ፡
+                                ለመተርጕማን፡ ጽድቃነ፡ ቃል።
+                              </incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.1.3">
+                              <locus target="#206r"></locus>
+                              <title type="complete" xml:lang="en">Colophon</title>
+                              <textLang mainLang="gez"></textLang>
+                              <incipit xml:lang="gez">
+                                ወዝኬ፡ ውእቱ፡ ትርጓሜ፡ አረጋዊ፡ ትሩፍ፡ ገብረ፡ እግዚአብሔር፡ <persName ref="PRS5503Ibnatt">አብ፡ ዮሐንስ፡ አፈ፡ ወርቅ፡
```

Comment:
Since this is a colophon of the work it would be good to add it as a textpart to the work record